modules/integration/tests-integration/tests-es/src/test/java/org/wso2/carbon/registry/es/notifications/CustomRXTStoreEmailNotificationTestCase.java

```diff
@@ -0,0 +1,289 @@
+*under the License.
+*/
+
+package org.wso2.carbon.registry.es.notifications;
+
+import org.apache.wink.client.ClientResponse;
+import org.json.JSONException;
+import org.json.JSONObject;
+import org.testng.annotations.*;
+import org.wso2.carbon.automation.engine.configurations.UrlGenerationUtil;
+import org.wso2.carbon.automation.engine.context.TestUserMode;
+import org.wso2.carbon.automation.engine.frameworkutils.FrameworkPathUtil;
+import org.wso2.carbon.automation.test.utils.common.TestConfigurationProvider;
+import org.wso2.carbon.identity.user.profile.stub.UserProfileMgtServiceUserProfileExceptionException;
+import org.wso2.carbon.identity.user.profile.stub.types.UserFieldDTO;
+import org.wso2.carbon.identity.user.profile.stub.types.UserProfileDTO;
+import org.wso2.carbon.integration.common.utils.exceptions.AutomationUtilException;
+import org.wso2.carbon.integration.common.utils.mgt.ServerConfigurationManager;
+import org.wso2.carbon.registry.core.exceptions.RegistryException;
+import org.wso2.carbon.registry.es.utils.EmailUtil;
+import org.wso2.carbon.registry.resource.stub.ResourceAdminServiceExceptionException;
+import org.wso2.greg.integration.common.clients.ResourceAdminServiceClient;
+import org.wso2.greg.integration.common.clients.UserProfileMgtServiceClient;
+import org.wso2.greg.integration.common.utils.GREGIntegrationBaseTest;
+import org.wso2.greg.integration.common.utils.GenericRestClient;
+
+import javax.activation.DataHandler;
+import javax.ws.rs.core.MediaType;
+import javax.xml.xpath.XPathExpressionException;
+import java.io.File;
+import java.io.IOException;
+import java.net.URL;
+import java.util.HashMap;
+import java.util.Map;
+
+import static org.testng.Assert.assertNotNull;
+import static org.testng.Assert.assertTrue;
+
+/**
+ * This test class can be used to check the email notification functionality on store side for custom asset type.
+ */
+public class CustomRXTStoreEmailNotificationTestCase extends GREGIntegrationBaseTest {
+
+    private TestUserMode userMode;
+    private String jSessionIdPublisher;
+    private String jSessionIdStore;
+    private UserProfileMgtServiceClient userProfileMgtClient;
+    private ResourceAdminServiceClient resourceAdminServiceClient;
+    private File axis2File;
+    private String publisherUrl;
+    private String storeUrl;
+    private String resourcePath;
+    private String assetId;
+    private String cookieHeaderPublisher;
+    private String cookieHeaderStore;
+    private GenericRestClient genericRestClient;
+    private Map<String, String> queryParamMap;
+    private Map<String, String> headerMap;
+    private String loginURL;
+    private String emailAddress;
+    private boolean isNotificationMailAvailable;
+
+
+    @Factory(dataProvider = "userModeProvider")
+    public CustomRXTStoreEmailNotificationTestCase(TestUserMode userMode) {
+        this.userMode = userMode;
+    }
+
+    @BeforeClass(alwaysRun = true)
+    public void init() throws Exception {
+
```

Comment:
Hi Fernando,
Thanks for pointing this out. I have removed all unnecessary white spaces from all the classes.
Please refer the PR #343 
Thanks